### ISSUE TITLE
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`

### ISSUE THREAD

**Issue by Jill0001** [vita-epfl/openpifpaf#248]
Hi, I'm trying to train a model with a pretrained modle.
I use the configs as below:
`time CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python3 -m openpifpaf.train \
  --lr=0.05 \
  --epochs=250 \
  --lr-decay 220 235 \
  --lr-decay-epochs=15 \
  --batch-size=8 \
  --square-edge=385 \
  --lambdas 1 1 0.2   1 1 1 0.2 0.2    1 1 1 0.2 0.2 \
  --auto-tune-mtl \
  --weight-decay=1e-5 \
  --update-batchnorm-runningstatistics \
  --checkpoint outputs/shufflenetv2k30w-200510-104256-cif-caf-caf25-o10s-0b5ba06f.pkl --extended-scale --orientation-invariant=0.1`

The pytorch version is 1.4.0
and I got this warning:
`/usr/local/lib/python3.6/dist-packages/torch/optim/lr_scheduler.py:122: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)`
What mistake did I make...?
Could you help me out? 
Thanks!

**Comment by Jill0001** [comment]
Seemingly this warning dose nothing harm to training

**Comment by svenkreiss** [comment]
Correct, you can ignore the warning. All the best for your project.
